pdns/pdns_recursor.cc

```diff
@@ -2986,6 +2981,10 @@ static int serviceMain(int argc, char*argv[])
     }
   }
 
+  SyncRes::parseEDNSSubnetWhitelist(::arg()["edns-subnet-whitelist"]);
+  SyncRes::parseEDNSSubnetAddFor(::arg()["ecs-add-for"]);
+  g_useIncomingECS = ::arg().mustDo("use-incoming-edns-subnet");
```

Comment:
The processing of `edns-subnet-whitelist` and `use-incoming-edns-subnet` is duplicated line 3033.

Reply:
Merge conflict resolution leftover that I've now removed. Thanks for noticing!